PostmanApiScripts/ChatApiTest.postman_collection.json

```diff
@@ -0,0 +1,201 @@
+{
+	"info": {
+		"_postman_id": "b7644feb-eb57-4e03-9456-14220fa757c2",
+		"name": "ChatApiTest",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "36242894"
```

Comment:
mostly looks good but you need to add schema validations for request and response for all   please see  https://www.softwaretestingmaterial.com/json-schema-validation-in-postman/